tests/input.rs

```diff
@@ -0,0 +1,15 @@
+use assert_cmd::Command;
+use predicates::prelude::predicate;
+
+#[test]
+fn input() -> Result<(), Box<dyn std::error::Error>> {
+    let mut cmd = Command::cargo_bin("cmjava")?;
+
+    cmd.arg("tests/data/input/Main.class");
+    cmd.write_stdin("A");
```

Comment:
style: To make it even more obvious that 65 is the ASCII representation of a, you could use a char A here (i.e. `let a = 'A';`) plus string conversions, i.e `format!("{}", a)` and `format!("{}\n", a as u8)`.